doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -83,7 +83,7 @@ described elsewhere
    Displays the type of :n:`@term`. When called in proof mode, the
    term is checked in the local context of the selected goal.
 
-.. cmd:: Search {+ @search_query } {? {| inside | outside } {+ @qualid } }
+.. cmd:: Search {+ @search_query } {? {| {| inside | outside } {+ @qualid } | in {+ @qualid } } }
```

Comment:
This should appear like this for readability:

```suggestion
.. cmd:: Search {+ @search_query } {? {| inside | in | outside } {+ @qualid } }
```

To do so, change the `ne_in_or_out_modules` lines in `common_edit.mlg` as follows:

```
ne_in_or_out_modules: [
| REPLACE "inside" LIST1 global
| WITH [ "in" | "inside" | "outside" ] LIST1 global
| DELETE "in" LIST1 global
| DELETE "outside" LIST1 global
]
```

Also, please describe "in" in the documentation with "inside"

![image](https://user-images.githubusercontent.com/1253341/140803468-025d0571-1969-444e-8ddd-29a4d7dadb61.png)

You could add another bullet point or use ``` :n:`{| inside | in }` ```